### ISSUE TITLE
where are you evaluating your model in each epochs?

### ISSUE THREAD

**Issue by IamExperimenting** [Pradhunmya/pytorch_faster_rcnn#1]
Hi @Pradhunmya , thanks very much for this content. It really helps us to understand. I wonder where have you implemented the evaluation step in the utlis.py script?

And I noticed you have created the same data loader variable name for the training dataset and test dataset? any rationale behind it or just a typo error?
![image](https://user-images.githubusercontent.com/74839416/135766142-848d7432-4e6d-44f5-b819-863f21995dbf.png)

**Comment by Pradhunmya** [comment]
Hi @IamExperimenting, 
I haven't added the evaluation step it's just we are doing it on the test set, and yes that was a typo, sorry about it.
